src/lib/tag-parsers/generate-page-auto-data.js

```diff
@@ -0,0 +1,43 @@
+import parseOgTags from './og-parser';
```

Comment:
`og:` is not the best regex to parse open graph tags

Reply:
for now we only care about product and breadcrumbs. So this implementation works.